charts/aws-efs-csi-driver/templates/controller-deployment.yaml

```diff
@@ -39,18 +48,26 @@ spec:
         {{- toYaml . | nindent 8 }}
         {{- end }}
       serviceAccountName: {{ .Values.controller.serviceAccount.name }}
-      priorityClassName: system-cluster-critical
+      priorityClassName: {{ .Values.controller.priorityClassName | default "system-cluster-critical" }}
       {{- with .Values.controller.tolerations }}
       tolerations: {{- toYaml . | nindent 8 }}
       {{- end }}
       {{- with .Values.controller.securityContext }}
       securityContext:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.controller.dnsPolicy }}
+      dnsPolicy: {{ .Values.node.dnsPolicy }}
+      {{- end }}
+      {{- with .Values.node.dnsConfig }}
```

Comment:
Why pull the dns policy + config from the node settings? 

I think this should be `.Values.controller.dnsPolicy` on line 60, and `.Values.controller.dnsConfig` on line 62.

Reply:
Once we address this, I think this is fine to merge

Reply:
Apologies for the delay in response. Incorporated the feedback under https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/1152/commits/0fe7e6ce7c0b21b117afc3b2f5d07a7fcaf6da40